dynamodb/src/it/scala/zio/dynamodb/LiveSpec.scala

```diff
@@ -225,6 +225,35 @@ object LiveSpec extends DefaultRunnableSpec {
               chunk  <- stream.runCollect
             } yield assert(chunk)(equalTo(Chunk(aviPerson, avi2Person, avi3Person)))
           }
+        },
+        testM("parallel scan all item") {
+          withTemporaryTable(
+            numberTable,
+            tableName =>
+              for {
+                _      <- batchWriteFromStream(ZStream.fromIterable(1 to 10000).map(i => Item(id -> i))) { item =>
+                            putItem(tableName, item)
+                          }.runDrain
+                stream <- scanAllItem(tableName).parallel(8).execute
+                chunk  <- stream.runCollect
```

Comment:
```suggestion
                count  <- stream.fold(0) { case (count, _) => count + 1 }
```
Really minor and totally optional suggestion to prevent reading everything into memory - same for test below